modules/portal-generate-init-bundle.adoc

```diff
@@ -44,41 +44,41 @@ You must have the `Admin` user role to create an init bundle.
 .Procedure
 
 ifndef::cloud-svc[]
-. Find the address of the {product-title-short} portal based on your exposure method:
-.. For a route:
-+
-[source,terminal]
-----
-$ oc get route central -n stackrox
-----
-.. For a load balancer:
+. Find the address of the {product-title-short} portal as described in "Verifying Central installation using the Operator method".
+endif::cloud-svc[]
+. Log in to the {product-title-short} portal.
+. If you do not have secured clusters, the *Platform Configuration* -> *Clusters* page appears.
+. Click *Create init bundle*.
+. Enter a name for the cluster init bundle.
+. Select your platform.
+. Select the installation method you will use for your secured clusters: *Operator* or *Helm chart*.
+. Click *Download* to generate and download the init bundle, which is created in the form of a YAML file.
```

Comment:
Aha, **in the form of a YAML file** is nice craftsmanship to distinguish the init bundle secrets in Central from the corresponding form of the init bundle for user.

Is it worthwhile to include a word like **reusable** somewhere, or to state explicitly as a concept that one init bundle and its corresponding YAML file for all secured clusters with a given installation method?